values/strings.xml

```diff
@@ -2127,6 +2129,7 @@ Cons: Less challenges</string>
     <string name="stage_account_steam">Steam</string>
     <string name="stage_gallery_update">Update</string>
     <string name="stage_gallery_noimages">No images uploaded, yet.</string>
+    <string name="stage_gallery_loadingimages">Loading images from the server...</string>
```

Comment:
Is that an actual string? If not, consider making a feature request.

Reply:
https://forum.theotown.com/ucp.php?i=pm&mode=view&p=27474